scripts/test_validate_renovate_ownership.py

```
@@ -0,0 +1,183 @@
+                source = ROOT / relative
+                target = root / relative
+                target.write_text(source.read_text(encoding="utf-8"), encoding="utf-8")
+            (root / ".github/workflows/test.yml").write_text(
+                "steps:\n  - uses: actions/checkout@" + "a" * 40 + " # v4\n", encoding="utf-8"
+            )
+            dockerfile = root / "images/base/Dockerfile"
+            dockerfile.write_text(
+                dockerfile.read_text(encoding="utf-8").replace("ARG UBUNTU_VERSION=", "ARG DISTRO_VERSION="),
+                encoding="utf-8",
+            )
+            with self.assertRaises(VALIDATOR.OwnershipError):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)